### ISSUE TITLE
Перерисовывается дельта после нажатия кнопки применить

### ISSUE THREAD

**Issue by NordonSAN** [SBProX/SBProX-Dev#1915]
Перерисовывается дельта после нажатия кнопки применить

Скрин до нажатия применить
![изображение](https://user-images.githubusercontent.com/52976306/111116464-a4b85880-8598-11eb-9825-97d0aa68af22.png)

Скрин после нажатия применить
![изображение](https://user-images.githubusercontent.com/52976306/111116537-b863bf00-8598-11eb-8de2-a01467abba99.png)

**Comment by aleksandr-exclusive** [comment]
И цвет баров тоже поменялся

<img width="810" alt="123" src="https://user-images.githubusercontent.com/72736701/111186463-b6f7be00-85bb-11eb-92f4-ded68992be53.png">

**Comment by NordonSAN** [comment]
Да, все очень серьезно, на всех таймфремах перерисовываются показатели. У меня после подключения к Квику последние два месяца полный раздрайв в торговле. Теперь понимаю из-за чего это. Когда данные брались всегда с сервера СБПро, то все было нормально. Теперь даже и не знаю как торговать, торговлю придется отложить пока не сделают СБПро или искать замену платформе, что не очень хочется, уже привык к ней
